user/build-stages.md

```diff
@@ -97,6 +97,25 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+### Naming your Jobs within Build Stages
+
+You can also name specific jobs within build stages. We recommend unique job names, but 
+do not enforce it (though this may change in the future). Job names are defined like other 
+job configuration elements within a build stage, but the value of the `name` must be in quotes.
```

Comment:
> Job names are defined ... within a build stage.

They're not really defined within a stage, but on the job, aren't they? 

> the value of the name must be in quotes

That's only true if the name has spaces, and ... as this is YAML ... there technically would be other ways to signify it as a string.

How about:

> Jobs defined in the `jobs.include` section can be given a name attribute as follows:

I'm also wondering if, after the example, we should mention that:

> Jobs coming out of the matrix expansion cannot be named.

Reply:
+1

Reply:
ah, thanks for catching the unclarity here... I knew what _I_ was thinking...